src/oslinfo/oslinfo.cpp

```diff
@@ -235,6 +244,7 @@ main (int argc, char *argv[])
                 "-h", &help, "Print help message",
                 "--help", &help, "",
                 "-v", &verbose, "Verbose",
+                "--bench", &benchmark, "Benchmark shader loading time",
```

Comment:
My only concerns are related to this line. I'm not sure it's is a dealbreaker, but I want to bring it up and see what you think.

I worry a bit that "benchmark shader loading time" might give the (incorrect!) impression that this is a reliable way to benchmark how long it would take the shader to load in ShadingSystem. Maybe if it is described as "benchmark OSLQuery read time of the shader" it would be more clear?

Looking at other command line tools within OSL (and OIIO), when there is an option to print timing or memory information about what it's doing, it's never called --bench, it's always called --stats or --runstats. (I apologize for the inconsistency there. These days I prefer --runstats to distinguish stats about the running program from stats about the input,)

A single change may address both of these issues. Calling it --runstats and describing it as "print run statistics (time, etc.)" might both be consistent with related tools as well as be very clear that all you're doing is timing the operations of this program, not necessarily shader load time in a renderer.